Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject.Tests/UnitTests.cs

```diff
@@ -0,0 +1,292 @@
+        private readonly string _clientID = Environment.GetEnvironmentVariable("Salesforce_ClientID");
+
+        private readonly string _domain = @"https://hiqfinlandoy2-dev-ed.my.salesforce.com";
+        private readonly string _username = "testuser@test.fi";
+        private readonly string _authurl = @"https://login.salesforce.com/services/oauth2/token";
+
+        private readonly CancellationToken _cancellationToken = new();
+        private Options _options;
+        private string _userJson;
+        private List<ResultObject> _result;
+
+        private string _name = "Test" + DateTime.Now.Year + "" + DateTime.Now.Month + "" + DateTime.Now.Day + "" + DateTime.Now.Hour + "" + DateTime.Now.Minute + "" + DateTime.Now.Millisecond;
+
+        #region helper classes
+        private class Account
+        { 
+            public string Name { get; set; }
+        }
+
+        private class Case
+        {
+            public string AccountId { get; set; }
+            public string Status { get; set; } = "New";
+            public string Origin { get; set; }
+            public string Description { get; set; }
+            public string Subject { get; set; }
+        }
+
+        private class Contact
+        {
+            public enum Salutation { Mr, Mrs, Ms, Dr, Prof }
+
+            public Salutation Title { get; set; }
+            public string LastName { get; set; }
+        }
+
+        private class ResultObject
+        { 
+            public string Type { get; set; }
+            public string Id { get; set; }
+        }
+        #endregion
+
+        [TestInitialize]
+        public async Task TestInitialize()
+        {
+            _result = new List<ResultObject>();
+
+            Account content = new Account
+            {
+                Name = _name
+            };
+            _userJson = JsonSerializer.Serialize(content);
+
+            _options = new Options
+            {
+                AuthenticationMethod = AuthenticationMethod.AccessToken,
+                AccessToken = await Salesforce.GetAccessToken(_authurl, _clientID, _clientSecret, _username, _password + _securityToken, _cancellationToken)
+            };
+        }
+#if true
+        [TestCleanup]
+        public async Task TestCleanUp()
+        {
+            if (_result != null)
+            {
+                for (var i = (_result.Count-1); i >= 0; i--) {
+                    var client = new RestClient(_domain + "/services/data/v54.0/sobjects/" + _result[i].Type + "/" + _result[i].Id);
+                    var request = new RestRequest("/", Method.Delete);
+
+                    request.AddHeader("Authorization", "Bearer " + _options.AccessToken);
+                    var response = await client.ExecuteAsync(request, _cancellationToken);
+                }
+                _result = null;
+            }
+        }
+#endif
+
+        [TestMethod]
+        public async Task TestCreateSObjects()
+        {
+            await AssertAccount();
```

Comment:
Each case should be in their own test methods, so please split these tests into separate test methods.